.github/workflows/pretest-and-test.yml

```diff
@@ -20,6 +20,15 @@ jobs:
       with:
         python-version: '3.8'
 
+    - uses: actions/cache@v3
+      with:
+        path: |
+          ~/.cache
+          ~/.onnx
```

Comment:
Question: What is this for?

Reply:
This seems to be a cache directory for onnx models from my personal stuff. Will remove it

Reply:
```suggestion
```